lib/features/login/domain/repository/authentication_oidc_repository.dart

```diff
@@ -11,4 +11,8 @@ abstract class AuthenticationOIDCRepository {
   Future<void> persistTokenOIDC(TokenOIDC tokenOidc);
 
   Future<TokenOIDC> getStoredTokenOIDC(String tokenIdHash);
+
+  Future<void> persistOidcConfiguration(OIDCConfiguration config);
```

Comment:
why not create other model to store `TokenOIDC` with its own `authority`? I propose it, because we don't need to `openBox` too much time.

Reply:
If you think `authority` should not be secured, we can use another storage method.

Reply:
no problem with the security, URL for `authServer` is public :).
I only keep the `Token` and `its authority` in the same place. Easy to manage, less operator to the db. That is my idea

Reply:
We will only save `authority` in `SharedPreference` 
```dart
  Future<void> persistAuthorityOidc(String authority);
```